lib/ript/dsl/primitives/filter.rb

```diff
@@ -80,7 +79,6 @@ def build_rule(name, block, opts={})
               to_address    = @labels[to][:address]
 
               attributes = {
-                             "table"       => "filter",
```

Comment:
filter is not understood on xenial, jump on a xenial box and try do iptables with --filter fail